eng/tools/sdk-suppressions/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@azure-tools/sdk-suppressions",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/index.js",
+  "version": "1.0.0",
+  "bin": {
+    "get-sdk-suppressions-label": "cmd/sdk-suppressions-label.js"
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose"
+  },
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "dependencies": {
+    "ajv": "^6.12.6",
+    "js-yaml": "^3.14.0",
```

Comment:
Can you convert this to use `yaml` instead of `js-yaml`, since we already have a dep on the former?

https://github.com/Azure/azure-rest-api-specs/blob/7c6c44b31926ef17e3956d15d03a4b1c148d03fa/eng/tools/suppressions/package.json#L19

Reply:
changed